stock_warehouse_flow/models/stock_warehouse_flow.py

```diff
@@ -342,14 +346,16 @@ def _generate_delivery_route(self):
                         }
                     )
                 wh_vals["out_type_id"] = self.to_picking_type_id.id
-                self.warehouse_id.write(wh_vals)
+                self.warehouse_id.with_context(do_not_check_quant=True).write(wh_vals)
```

Comment:
I know this change is coming from 14.0, but having a comment explaining the reason of this context key is very welcome here (in another commit to be easily backported if needed).
Note: for new commits better to avoid writing the version number in the message, it makes sense only for migration commit ("Migration to X.Y") but others commits could be backported/forward ported across versions :) Thanks!

Reply:
No problem!